app/controllers/admin/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::UsersController < ApplicationController
   before_action :authorize_admin!
   before_action :authorize_super_admin!, only: [ :update, :destroy ]
+  after_action :remove_user_locks, only: [:destroy]
```

Comment:
that sounds like a model concern / if it is done in a different controller we would still want it to be done